lib/livebook/notebook.ex

```diff
@@ -69,6 +71,14 @@ defmodule Livebook.Notebook do
               url: String.t()
             }
 
+  @type output_panel_row :: %{items: list(output_panel_item)}
+
+  @type output_panel_item :: %{
+          id: Livebook.Utils.id(),
+          width: 1..12,
+          outputs: list(Cell.indexed_output())
```

Comment:
I added the cell outputs here. In a previous comment you mentioned that this can't be saved in the livemd file but since we also save the cell_id here which is not savable either, I think it is ok for now?

Reply:
I wouldn't keep it here, because we then we need to synchronize all changes to cell outputs here. It's better to get with the minimal information we need, which is the `cell_id`. We can then grab the corresponding outputs perhaps in `data_to_view`, making sure they are up to date.

Reply:
Ahhh I see :)